notebooker/utils/results.py

```diff
@@ -97,7 +97,7 @@ def get_latest_successful_job_results(
 
 
 def get_all_result_keys(
-    serializer: NotebookResultSerializer, limit: int = 0, force_reload: bool = False
+    serializer: MongoResultSerializer, limit: int = 0, force_reload: bool = False
```

Comment:
Is this type correct? Maybe there should be a base class or interface all serializers have to import?